posturebest/Components/ConfigureAlertsTab.swift

```diff
@@ -13,6 +13,8 @@ struct ConfigureAlertsTab: View {
     @State private var standingFreqState: String = "15"
     @State private var postureFreqState: String = "5"
     let postureOpts = ["5", "10", "15", "20", "25", "30"]
+    let postureInfo = "This configuration will set how often the vest will alert you that your posture is not optimal."
+    let standingInfo = "This configuration will set how often the vest will alert you to stand up to improve movement throughout the day."
```

Comment:
Question on organization. Why do this here as a variable? I understand the premise in theory, but I don't know if it helps in the long run. If we have 50 different spots for text/messages, eventually wouldn't there just be a bunch of variables defined in a row here? I think it might just be clearer to see what the button should say in the same line as the button is defined on the page. Does that make sense?

Reply:
yeah I get what you're saying, I was thinking that having large blocks of text in the middle if the code would make it harder to read and work on in the future, and that it would make the actual org of the page more clear. If you still think its best to have in line im fine with it, lmk!

Reply:
I think we are good for now! I think it might be worth changing if we add a ton more and it starts to get confusing with what text is for what component.